_ml-commons-plugin/algorithms.md

```diff
@@ -316,4 +400,218 @@ The API responds with the sum of the contribution and base values per aggregatio
 
 ### Limitations
 
-The Localization algorithm can only be executed directly. Therefore, it cannot be used with the ML Commons Train and Predict APIs.
\ No newline at end of file
+The Localization algorithm can only be executed directly. Therefore, it cannot be used with the ML Commons Train and Predict APIs.
+
+## Logistic regression
+
+A classification algorithm, logistic regression models the probability of a discrete outcome given an input variable. In ML commons, these classifications include both binary and multi-class. The most common is the binary classification, which takes two values such as "true/false" or "yes/no", and predicts the outcome based on the values specified. On the other hand, a multi-class output can categorize different inputs based on type. This makes logistic regression most useful for situations where you are trying to determine how your inputs fit best into a specified-category. 
+
+### Parameters
+
+| Parameter | Type | Description | Default Value |
+|---|---|---|---|
+| learningRate | Double | The rate of speed the gradient moves during gradient descent | 1 |
```

Comment:
Is this the "Gradient descent step size"?

Reply:
Yes it's the step size at each iteration while moving toward a minimum of a loss function or optimal value.

Reply:
Thank you, @Zhangxunmt. This is a better way to explain the parameter.